python_modules/dagster/dagster/_core/definitions/asset_key.py

```diff
@@ -39,29 +45,31 @@ class AssetKey(NamedTuple("_AssetKey", [("path", PublicAttr[Sequence[str]])])):
             strings represent the hierarchical structure of the asset_key.
     """
 
-    def __new__(cls, path: Union[str, Sequence[str]]):
+    # Originally AssetKey contained "path" as a list. In order to change to using a tuple, we now have
+    parts: Sequence[str]  # with path available as a property defined below still returning a list.
```

Comment:
i went with "parts" as that made sense to me but can change to `path_tuple` or whatever someone feels strongly about

Reply:
parts sgtm